server/fishtest/templates/tests_view.mak

```diff
@@ -860,3 +860,27 @@
     % endif
     });
 </script>
+<script>
+  document.getElementById("gamesDownloadButton")?.addEventListener("click", function () {
+    async function downloadFile() {
+      try {
+        const response = await fetch(`/api/run_pgns/${run["_id"]}.pgn.gz`);
+        if (!response.ok) {
+          const errorText = await response.text();
+          throw new Error(errorText || "Error fetching file");
+        }
+        const blob = await response.blob();
+        const url = window.URL.createObjectURL(blob);
+        const a = document.createElement("a");
+        a.href = url;
+        a.download = `${run["_id"]}.pgn.gz`;
+        document.body.appendChild(a);
+        a.click();
+        document.body.removeChild(a);
+      } catch (error) {
+        alert(error.message);
```

Comment:
we don't use alert, as it is non css customisable (not part of the DOM) and quite old way to alert something to the user
probably console.error is better here.